### PR TITLE
Remove test helpers (small changes #2)

### PR DESCRIPTION
**Pull request by FK78**
Resolves #3264

**Overall change:** _Removed test helper imports._

**Code changes:**

- _Imports that do not change snapshot coverage significantly have been imported._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/PageHandlers/withError/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+exports[`withError HOC should return the errorMain component 1`] = `
+<h1>
+  This is an error.
+</h1>
 `;
 
-exports[`withError HOC with no error should return the passed in component 1`] = `<Component />`;
+exports[`withError HOC should return the passed in component 1`] = `
+<h1>
+  500
```
Comment (thekp) [inline src/app/containers/PageHandlers/withError/__snapshots__/index.test.jsx.snap]: 500 suggests that it is an error.

File: src/app/containers/PageHandlers/withError/index.test.jsx [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
-import { shouldShallowMatchSnapshot } from '../../../../testHelpers';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import WithError from '.';
 
+// eslint-disable-next-line react/prop-types
+jest.mock('../../ErrorMain', () => () => <h1>This is an error.</h1>);
+
 describe('withError HOC', () => {
-  const Component = () => <h1>Hola</h1>;
+  const Component = () => <h1>500</h1>;
```
Comment (thekp) [inline src/app/containers/PageHandlers/withError/index.test.jsx]:
```suggestion
  const Component = () => <h1>Hola</h1>;
```

Comment (thekp) [reply src/app/containers/PageHandlers/withError/index.test.jsx]: you will have to update snapshots

File: src/app/containers/PageHandlers/withError/index.test.jsx [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
   const ErrorHOC = WithError(Component);
 
-  describe('with error', () => {
-    shouldShallowMatchSnapshot(
-      `should return the errorMain component`,
-      <ErrorHOC error={new Error('This is an error.')} />,
-    );
-  });
+  shouldMatchSnapshot(`should return the passed in component`, <ErrorHOC />);
```
Comment (thekp) [inline src/app/containers/PageHandlers/withError/index.test.jsx]:
```suggestion
  shouldMatchSnapshot('should return the passed in component', <ErrorHOC />);
```

File: src/app/containers/PageHandlers/withError/index.test.jsx [hunk 4]
```diff
@@ -1,22 +1,18 @@
...
 
+// eslint-disable-next-line react/prop-types
+jest.mock('../../ErrorMain', () => () => <h1>This is an error.</h1>);
+
 describe('withError HOC', () => {
-  const Component = () => <h1>Hola</h1>;
+  const Component = () => <h1>500</h1>;
   const ErrorHOC = WithError(Component);
 
-  describe('with error', () => {
```
Comment (thekp) [inline src/app/containers/PageHandlers/withError/index.test.jsx:9]: We could keep the `describe` here so that it is clear what we are testing.

**Review by thekp — COMMENTED** [review]
just a few little changes for clarity 👍

**Review by dr3 — APPROVED** [review]

**Review by thekp — APPROVED** [review]
thanks for the changes

**Pull request merged**